CHANGELOG.md

```diff
@@ -66,6 +84,7 @@ Read more about migrating a plugin from Piwik 2.X to Piwik 3 on our [Migration g
 * The JavaScript AjaxHelper has a new method `ajaxHelper.withTokenInUrl()` to easily send a token along a XHR. Within the Controller the existence of this token can be checked via `$this->checkTokenInUrl();` to prevent CSRF attacks.
 * The new class `Piwik\Updater\Migration\Factory` lets you easily create migrations that can be executed during an update. For example database or plugin related migrations. To generate a new update with migrations execute `./console generate:update`.
 * The new method `Piwik\Updater::executeMigration` lets you execute a single migration.
+* New event `ViewDataTable.filterViewDataTable` let's you filter available visualizations
```

Comment:
I removed the old `ViewDataTable.addViewDataTable` and replaced it with `ViewDataTable.filterViewDataTable`. In general we always want to add components via the PHP files and to filter things we always use events. The `ViewDataTable.filterViewDataTable` is used and needed by Treemap plugin but will be also useful for other plugins